### PR TITLE
Reorganised where projects are saved and added defaultCRS information

### PR DESCRIPTION
**Pull request by OsiJr**
Note that for this to work fully it requires VDK0.4.1 due to a bug this exposed in root node metadata

### CODE REVIEW

File: src/vcProject.cpp [hunk 1]
```diff
@@ -129,6 +133,50 @@ void vcProject_Deinit(vcState *pProgramData, vcProject *pProject)
...
+      exportFilename.SetFromFullPath("%s/untitled_project.json", pPath);
+    else if (exportFilename.HasFilename())
+      exportFilename.SetExtension(".json");
+    else
+      exportFilename.SetFilenameWithExt("untitled_project.json");
+
+    // Check if file path exists before writing to disk, and if so, the user will be presented with the option to overwrite or cancel
+    if (allowOverride || vcModals_OverwriteExistingFile(exportFilename.GetPath()))
+    {
+      vcState::ErrorItem projectError;
```
Comment (peterAdams1014) [inline src/vcProject.cpp]: Potential uninitialized memory

File: src/vcProject.cpp [hunk 2]
```diff
@@ -129,6 +133,50 @@ void vcProject_Deinit(vcState *pProgramData, vcProject *pProject)
...
+        projectError.resultCode = udR_Success;
+      else
+        projectError.resultCode = udR_WriteFailure;
+
+      pProgramState->errorItems.PushBack(projectError);
+
+      vcModals_OpenModal(pProgramState, vcMT_ProjectChange);
+    }
+
+    if (pDir != nullptr)
```
Comment (peterAdams1014) [inline src/vcProject.cpp]: Unnecessary `nullptr` check.

File: src/vcProject.cpp [hunk 3]
```diff
@@ -129,6 +133,50 @@ void vcProject_Deinit(vcState *pProgramData, vcProject *pProject)
...
+      udFree(pDir);
```
Comment (DamianMadden) [inline src/vcProject.cpp]: nullptr check unnecessary, also udCloseDir?

File: src/vcProject.cpp (hunk 1 above)
Comment (OsiJr) [reply src/vcProject.cpp]: This is currently just moved code- do you want me to also make fixes (which complicates this MR)?

**Pull request merged**